src/include/fips.h

```diff
@@ -20,6 +20,14 @@
 extern int fips;			/* module status */
 
 
+/*
+ * Perform an integrity check on libica.so by calculating an HMAC from
+ * the file contents using a static HMAC key, and comparing it to a
+ * pre-calculated HMAC in a separate file. The HMAC key and HMAC file
+ * may be provided by a Distributor when building the packet.
+ */
+void fips_lib_integrity_check(void);
```

Comment:
since you put the prototype in fips.h, id move the corresponding function to fips.c (from fips_check.c).

Reply:
see answer above: no problem to add all code from fips_check.c to fips.c, or just only fips_lib_integrity_check. However, if we decide to move fips_lib_integrity_chec to fips.c then I would also move the rest.

Reply:
Id put everything in fips.c/.h

Reply:
Done.